src/platform/packages/shared/kbn-apm-synthtrace/src/cli/utils/start_live_data_upload.ts

```diff
@@ -91,6 +96,22 @@ export async function startLiveDataUpload({
     }
   }
 
+  function startPeriodicPerfLogging() {
+    let cpuUsage = process.cpuUsage();
+
+    setInterval(() => {
```

Comment:
this keeps the process open no?

Reply:
yes. this needs to be cleaned.

Reply:
[14864ea](https://github.com/elastic/kibana/pull/216686/commits/14864ea1572e0025e3f5c1b7d19b91dc8ed2942d)